pkg/noderesourcetopology/least_numa_test.go

```diff
@@ -0,0 +1,271 @@
+package noderesourcetopology
```

Comment:
same, let's add the copyright boilerplate

Reply:
:+1: